### PR TITLE
fix package import - should only be imported when there are transactions

### PR DESCRIPTION
**Pull request by dxu429**
https://clever.atlassian.net/browse/DD-5011

This import should only be generated when there are transactions in any table. This was introduced in the lastest version: https://github.com/Clever/wag/pull/473

- [x] Run `make build`
- [x] Run `make generate`
- [x] Update the current version in the `/VERSION` file.

# Testing
with v9.1.4, the import included but was unused which caused build errors

Screenshot 2024-03-04 at 3 45 21 PM
Screenshot 2024-03-04 at 3 46 48 PM

with this fix: 
Screenshot 2024-03-04 at 1 31 10 PM

### CODE REVIEW

File: server/gendb/bindata.go [hunk 1]
```diff
@@ -109,7 +109,7 @@ func dynamodbGoTmpl() (*asset, error) {
 	}
 
 	info := bindataFileInfo{name: "dynamodb.go.tmpl", size: 0, mode: os.FileMode(0), modTime: time.Unix(0, 0)}
-	a := &asset{bytes: bytes, info: info, digest: [32]uint8{0xc0, 0xa3, 0xce, 0x3d, 0xab, 0xfc, 0x2, 0xea, 0x20, 0x3, 0x0, 0x1e, 0xb, 0x26, 0x27, 0x54, 0x67, 0xb5, 0xef, 0x2f, 0x52, 0x94, 0x9d, 0x97, 0xed, 0x90, 0x3, 0x9d, 0xa4, 0x52, 0xe3, 0x50}}
+	a := &asset{bytes: bytes, info: info, digest: [32]uint8{0x5a, 0x84, 0x42, 0xe8, 0x33, 0xfe, 0x9, 0x3, 0x22, 0xfb, 0x16, 0xa3, 0x5a, 0xad, 0xd6, 0x3d, 0x98, 0xa, 0x70, 0xb3, 0x41, 0x4b, 0xf8, 0x39, 0xf3, 0x7, 0xc6, 0xba, 0xd4, 0x2e, 0x2a, 0xcf}}
```
Comment (sophlanz) [inline server/gendb/bindata.go:112]: Question: What is this change doing? Thanks in advance!

File: server/gendb/bindata.go [hunk 2]
```diff
@@ -93,7 +93,7 @@ func dynamodbLocalShTmpl() (*asset, error) {
...
...f5\x14\x79\x6a\x1e\x9a\xc7\x05\x86\x9c\xad\x3b\x4c\xb6\xd6\xed\x6d\x3b\x38\x15\xd7\x76\x8f\x8c\x74\xb3\xd3\x07\x9e\xe0\xad\xfd\xf4\x60\x18\x82\xfb\x06\xc4\x9b\x31\x83\xb4\x3e\xca\x3f\x4a\xa8\x3a\x66\x78\x82\x1e\xdd\x1c\x86\x23\xef\xb5\x54\x09\x33\x43\xbb\xf6\xfe\xf5\xd5\xcb\x97\x2f\xff\x31\x02\xdf\x2f\xbf\xba\x80\x12\x29\xef\x58\x9c\x21\x48\x11\xaf\x41\x67\x69\x2a\x95\xd1\x90\x73\x80\x42\x2d\xe3\x8c\xcc\x1e\x74\x5b\x77\x63\xd4\x30\x84\xf3\x93\x2c\x3c\xff\x4a\x26\x3e\xe8\x73\x1f\xeb\xd7\x9c\x12\x86\x76\xe9\xc7\xfb\x54\x0d\x83\xbe\xe9\x35\x3c\x77\x6e\x8e\x21\x61\xe9\xaf\xee\xfa\xb7\xf3\xc2\x05\xaf\x6c\x59\x1f\xc9\x85\x3a\x51\xc1\x58\x76\xd7\x67\x77\x4c\x51\x69\x59\x13\x20\xbf\xef\x56\xc9\x86\x8f\x2c\xd6\x7d\xb4\xec\x38\x5c\x9f\x6c\xea\x1c\x3c\x73\xaf\x91\x0b\x17\x6b\x0f\xe3\x89\xb1\x2c\xc8\xfc\xd9\x7e\x25\x04\xef\x70\x95\xdf\x1d\x8e\x3c\xaa\x81\x32\x26\xc3\xf3\x9d\xc8\x91\x67\x89\x86\x23\x2b\x32\x7f\x49\x50\x53\x53\x7f\xc1\xbe\xfb\xe3\x5e\x02\xb8\xd7\x00\xb0\x8b\xc6\xa4\x6a\xd7\x6e\x1f\x72\x05\x65\x78\xea\x64\x36\x1c\xba\x42\xe3\x02\x52\x27\xb2\x6b\xc3\xd1\xe0\x59\xe5\x5d\x43\xa1\x76\x5c\x4a\x1e\xef\xf8\x8b\x97\x65\x95\xc2\xfd\x5f\x00\x00\x00\xff\xff\xc5\x07\x39\x24\x4a\x27\x00\x00")
```
Comment (sophlanz) [inline server/gendb/bindata.go:96]: Curious about this change as well. Thanks!

**Review by sophlanz — APPROVED** [review]
LGTM 🎉 ! Just left a couple of questions about some of the generated code. I assume it was re-generated after adding a check to the config for anyTableHasTransactionsEnabled, but curious. Thanks for providing screenshots!

File: server/gendb/bindata.go (hunk 1 above)
Comment (dxu429) [reply server/gendb/bindata.go:112]: I believe it is the binary embedding for the template. You can read more about go-bindata in [this repo](https://github.com/go-bindata/go-bindata)

Comment (sophlanz) [reply server/gendb/bindata.go:112]: Thank you!! @dxu429

**Pull request merged**